src/FffGcodeWriter.cpp

```diff
@@ -33,13 +33,14 @@ FffGcodeWriter::FffGcodeWriter()
 
 void FffGcodeWriter::writeGCode(SliceDataStorage& storage, TimeKeeper& time_keeper)
 {
-    gcode.preSetup(getStartExtruder(storage));
+    size_t start_extruder_nr = getStartExtruder(storage);
```

Comment:
Make this const to avoid it getting changed later by mistake?

Reply:
Good point.